### ISSUE TITLE
Clipboard feature does not work in Remoteapp mode

### ISSUE THREAD

**Issue by FranckGoirand** [FreeRDP/FreeRDP#1636]
Hi
i test freerdp in remoteapp mode with Windows 2008 R2, but clipboard does not work in this mode.
In remote desktop, it's ok.

my command line:
./client/X11/xfreerdp +clipboard /u:test /p:test /w:1366 /h:768 /app:\"%ProgramFiles%/webkiosk/word.bat\" /v:192.168.1.1

> git log
> commit 72f38ce75a86095b831a01cd3606ec77b81dd444
> Merge: 0ed9936 8767931
> Author: Marc-André Moreau marcandre.moreau@gmail.com
> Date:   Fri Dec 13 08:59:28 2013 -0800

```
Merge pull request #1635 from hardening/capa_fixes

Harden capability parsing
```

Thanks for your help

**Comment by meersjo** [comment]
Just tested with current master (c325ec532) - still does not work.

**Comment by bmiklautz** [comment]
Should be fixed with #3934
